rosdep/python.yaml

```diff
@@ -7466,6 +7466,20 @@ python3-pytest:
       packages: [pytest]
   rhel: ['python%{python3_pkgversion}-pytest']
   ubuntu: [python3-pytest]
+python3-pytest-asyncio:
+  alpine: [py3-pytest-asyncio]
+  arch: [python-pytest-asyncio]
+  debian: [python3-pytest-asyncio]
+  fedora: [python3-pytest-asyncio]
+  gentoo: [dev-python/pytest-asyncio]
+  nixos: [python3Packages.pytest-asyncio]
+  opensuse: [python3-pytest-asyncio]
+  osx:
+    pip:
+      packages: [pytest-asyncio]
+  ubuntu:
```

Comment:
```suggestion
  rhel:
    '*': [python3-pytest-asyncio]
    '7': null
  ubuntu:
```

Reply:
Thanks, I never know where to look for rhel packages.
Is there any reason why `opensuse` and `rhel` are not in the `Links to Distribution Packages` section of the PR template?

Reply:
I don't think there is a reason, no.